BackofficeBundle/Resources/public/coffee/tinyMCE/activateTinyMCE.coffee

```diff
@@ -1,7 +1,22 @@
 # ACTIVATE TINYMCE
+
 callback_tinymce_init = null
 
+doCallBack = (editor, view) ->
```

Comment:
@nicolasThal : I have read your question too fast and did a half-answer. This callback is empty because it is called in this cms-bundle file for a job required in media-admin. So it does nothing in cms-bundle but it is overriden is media-admin.
I'm not fan of the way it works because it allows only one override for all the bundles, but as it was here before my work and as I have no alternative for the moment, I let this like that, but I suggest a refacto when we'll have a better way of doing this.